### PR TITLE
Add ojo testnet

### PR DESCRIPTION
**Pull request by adamewozniak**

### CODE REVIEW

File: testnets/ojotestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "sauron",
```
Comment (JeremyParish69) [inline testnets/ojotestnet/assetlist.json:3]: please set chain_name to ojotestnet (assumng mainnet will have chain_name ojo)

Comment (adamewozniak) [inline testnets/ojotestnet/assetlist.json:3]:
```suggestion
    "chain_name": "ojotestnet",
```

File: testnets/ojotestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "../../chain.schema.json",
+    "chain_name": "sauron",
```
Comment (JeremyParish69) [inline testnets/ojotestnet/chain.json:3]: sorry, this one, too. To 'ojotestnet'

Comment (adamewozniak) [inline testnets/ojotestnet/chain.json:3]:
```suggestion
    "chain_name": "ojotestnet",
```

**Pull request merged**